### DISCUSSION TITLE
Can't add ebooks

### DISCUSSION THREAD

**Discussion by jakemelon** [iiab/iiab#3762] [Q&A]
Hi 

Successfully installed iiab on Pi 5 booting from 2tb nvme/hat. All works well except for:

1. I cannot find any documentation on how to add 100 or so ebooks to my iiab (from USB thumb drive to my boot drive)  There is no option in iiab anywhere to add ebooks content.
2.  When I go to calibre-web on my iiab there is no way to access it's admin interface. It asks for user id and it is not taking my raspberry user id nor my iiab admin one. Where do you define that? 

Thank you!
Jake

**Comment by holta** [comment]
Hi @jakemelon !

Have you reviewed IIAB's Calibre-Web documentation?

- https://github.com/iiab/iiab/blob/master/roles/calibre-web/README.rst
- https://github.com/iiab/calibre-web/wiki

Does that help?
